recipe-server/normandy/recipes/models.py

```diff
@@ -560,7 +560,7 @@ def get_absolute_url(self):
         return reverse('action-detail', args=[self.name])
 
     def compute_implementation_hash(self):
-        return hashlib.sha1(self.implementation.encode()).hexdigest()
+        return sri_hash(self.implementation.encode(), url_safe=True)
```

Comment:
`sri_hash` is specifically for the purposes of subresource integrity, but it's being used here as a generic hash util. If we want a helper util for this, it should be separate from `sri_hash`, and maybe `sri_hash` should depend on it instead.